intertidal/mosaics.py

```diff
@@ -0,0 +1,193 @@
+import s3fs
+import shutil
+import os
+import glob
+import sys
+from pathlib import Path
+import tempfile
+import subprocess
+import pandas as pd
+import geopandas as gpd
+from pathlib import Path
+from urllib.parse import urlparse
+import click
+
+from datacube.utils.aws import configure_s3_access
+from intertidal.utils import (
+    configure_logging,
+    round_date_strings,
+)
+from pathlib import Path
+import os
+
+
+def find_files(directory, pattern):
```

Comment:
This doesn't seem to be used in the code

Reply:
It is not. I will remove it. :)